docker/bedtools/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:22.04 AS builder
+
+RUN apt-get update && apt-get install -y build-essential zlib1g-dev libbz2-dev liblzma-dev wget python3
+
+RUN wget https://github.com/arq5x/bedtools2/releases/download/v2.31.1/bedtools-2.31.1.tar.gz \
+    && tar -xzf bedtools-2.31.1.tar.gz \
+    && cd bedtools2 \
+    && make static \
```

Comment:
This is the key difference and why I am manually building `bedtools`. This creates a statically linked binary that I can subsequently run in the `alpine` image.

Reply:
"key difference" as opposed to what? The BioContainers image I'm guessing?

Reply:
That or downloading the release binaries.